Documentation/components/nxwidgets.rst

```diff
@@ -43,17 +43,11 @@ Note: Many of the fundamental classed in NxWidgets derive from the
 Antony Dzeryn's "Woopsi" project: http://woopsi.org/ which also has a
 BSD style license. See the COPYING file for details.
 
-NXWidgets DOxygen Documentation
+NXWidgets Doxygen Documentation
 ===============================
 
-Release notes, DOxygen documentation, as well as downloads for the
-latest NxWidgets releases are available online:
-
-**NxWidgets-1.19**:
-`Documentation <http://nuttx.org/nxwidgets_v1_19/html/index.html>`__,
-`Release
-notes <http://sourceforge.net/projects/nuttx/files/NxWidgets/NxWidgets-1.19/>`__,
-and
-`Downloads <http://sourceforge.net/projects/nuttx/files/NxWidgets/NxWidgets-1.19/>`__
+.. note::
```

Comment:
Please use "todo" admonition for notes intended to NuttX maintainers. The rest of admonitions are to address readers of the documentation.